apps/rush/src/utilities/InstallManager.ts

```diff
@@ -384,9 +382,79 @@ export default class InstallManager {
         }
       }
 
-      // Don't update the file timestamp unless the content has changed, since "rush install"
-      // will consider this timestamp
-      JsonFile.saveJsonFile(tempPackageJson, tempPackageJsonFilename, { onlyIfChanged: true });
+      // NPM expects the root of the tarball to have a directory called 'package'
+      const npmPackageFolder: string = 'package';
+
+      // Example: "C:\MyRepo\common\temp\projects\my-project-2.new"
+      const tempProjectFolder: string = path.join(
+        path.dirname(rushProject.tempPackageTarballFilename),
+        unscopedTempProjectName + '.new');
+
+      // Example: "C:\MyRepo\common\temp\projects\my-project-2\package.json"
+      const tempPackageJsonFilename: string = path.join(tempProjectFolder, RushConstants.packageJsonFilename);
+
+      // Example: "C:\MyRepo\common\temp\projects\my-project-2.gzip"
+      const tarballFile: string = rushProject.tempPackageTarballFilename;
+
+      // Example: "C:\MyRepo\common\temp\projects\my-project-2.old"
+      const extractedFolder: string = tempProjectFolder + '.old';
+
+      // we only want to overwrite the package if the existing tarball's package.json is different from tempPackageJson
+      let shouldOverwrite: boolean = true;
+      try {
+        // extract the tarball and compare the package.json directly
+        if (fsx.existsSync(tarballFile)) {
+
+          // ensure the folder we are about to extract into is clean
+          fsx.removeSync(extractedFolder);
+          fsx.mkdirpSync(extractedFolder);
```

Comment:
This fails intermittently due the virus scanner.

See Utilities.createFolderWithRetry() for more info